mc2/tasks.py

```diff
@@ -1,7 +1,7 @@
-from celery import task
+from mc2 import celery_app
```

Comment:
This module doesn't seem to exist. Should `mc2/celery.py` be `mc2/celery_app.py` instead?